<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PropertiesDialog</class>
 <widget class="QDialog" name="PropertiesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>597</width>
    <height>389</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Propiedades de exportación</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_7">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Directorio a exportar:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leDirectorio">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButton">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Básico</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="QCheckBox" name="chkORM">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Generar clases ORM</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Nombre de Directorio:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="leORM"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QCheckBox" name="chkDoc">
          <property name="text">
           <string>Generar documentación</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Nombre de Directorio:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="leDoc">
            <property name="enabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QCheckBox" name="chkIG">
          <property name="text">
           <string>Generar Interfaces Gráficas</string>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Nombre de Directorio:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="leIG">
            <property name="enabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_7">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Prefijo para GUI's</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lePrefijo">
            <property name="enabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Extras</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_8">
        <item>
         <widget class="QCheckBox" name="chkIntegrar">
          <property name="text">
           <string>Integrar la documentación con QtHelp(*.qhcp)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="chkFull">
          <property name="text">
           <string>Generar proyecto completo</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Nombre</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="leTarget">
            <property name="enabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>PropertiesDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>216</x>
     <y>371</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkDoc</sender>
   <signal>toggled(bool)</signal>
   <receiver>leDoc</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>127</x>
     <y>158</y>
    </hint>
    <hint type="destinationlabel">
     <x>144</x>
     <y>215</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkDoc</sender>
   <signal>toggled(bool)</signal>
   <receiver>label_2</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>219</x>
     <y>158</y>
    </hint>
    <hint type="destinationlabel">
     <x>194</x>
     <y>189</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkORM</sender>
   <signal>toggled(bool)</signal>
   <receiver>label_4</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>81</x>
     <y>77</y>
    </hint>
    <hint type="destinationlabel">
     <x>91</x>
     <y>108</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkORM</sender>
   <signal>toggled(bool)</signal>
   <receiver>leORM</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>61</x>
     <y>77</y>
    </hint>
    <hint type="destinationlabel">
     <x>75</x>
     <y>134</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkIG</sender>
   <signal>toggled(bool)</signal>
   <receiver>label_3</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>201</x>
     <y>225</y>
    </hint>
    <hint type="destinationlabel">
     <x>169</x>
     <y>251</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkIG</sender>
   <signal>toggled(bool)</signal>
   <receiver>leIG</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>108</x>
     <y>232</y>
    </hint>
    <hint type="destinationlabel">
     <x>109</x>
     <y>270</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkIG</sender>
   <signal>toggled(bool)</signal>
   <receiver>label_7</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>91</x>
     <y>223</y>
    </hint>
    <hint type="destinationlabel">
     <x>110</x>
     <y>304</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkIG</sender>
   <signal>toggled(bool)</signal>
   <receiver>lePrefijo</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>235</x>
     <y>239</y>
    </hint>
    <hint type="destinationlabel">
     <x>222</x>
     <y>313</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkFull</sender>
   <signal>clicked(bool)</signal>
   <receiver>leTarget</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>352</x>
     <y>92</y>
    </hint>
    <hint type="destinationlabel">
     <x>385</x>
     <y>123</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
